telecine/src/main/AndroidManifest.xml

```diff
@@ -47,6 +47,16 @@
         android:taskAffinity=""
         android:exported="true"
         />
+
+    <service
+        android:icon="@drawable/ic_videocam_white_24dp"
+        android:label="@string/app_name"
```

Comment:
This should probably be `@string/launch_overlay` to match the shortcut.

I'm also wondering if it would be beneficial to have a version of the launcher icon that can be used here.